src/auth/views/Login.tsx

```diff
@@ -44,6 +46,7 @@ const LoginView: React.FC<LoginViewProps> = ({ params }) => {
 
     if (errors.length === 0) {
       setLastLoginMethod("password");
+      setEmailUsedToLogin(data.email);
```

Comment:
shouldnt we prompt user to save login/email? Feels wrong if using public computer, idk